### PR TITLE
Fix for azuresynapse.net cursor color

### PR DESCRIPTION
**Pull request by DaveRuijter**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1535,6 +1535,13 @@ CSS
 
 ================================
 
+azuresynapse.net
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1538]: Is this a internal website?

Comment (DaveRuijter) [reply src/config/dynamic-theme-fixes.config:1538]: No, it is the portal of the suite for the Microsoft Azure Synapse Analytics cloud offering.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1538]: Is it only reachable by a subdomain?

Comment (DaveRuijter) [reply src/config/dynamic-theme-fixes.config:1538]: I think it is yes. Full url is: https://web.azuresynapse.net/

**Review by Gusted — APPROVED** [review]

**Pull request merged**